dk-installer.py

```diff
@@ -1154,19 +1180,20 @@ class DemoContainerAction(Action):
 
     def run_dk_demo_container(self, command: str):
         with self.start_cmd(
-            "docker",
-            "run",
-            "--rm",
-            "--mount",
-            f"type=bind,source=.{os.path.sep}{DEMO_CONFIG_FILE},target=/dk/{DEMO_CONFIG_FILE}",
-            "--name",
-            DEMO_CONTAINER_NAME,
-            "--network",
-            DOCKER_NETWORK,
-            "--add-host",
-            "host.docker.internal:host-gateway",
-            DEMO_IMAGE,
-            command,
+                "docker",
+                get_docker_compose_path(),
```

Comment:
if I understood the logic correct, this will crash on linux, because `get_docker_compose_path()` would return `None`

Why the extra indentation level?